lib/spell-check-rules.js

```diff
@@ -13,6 +13,10 @@ module.exports = [
 		test: /\bjavascript\b/gi,
 		value: 'JavaScript'
 	},
+	{
+		test: /\bjavascript\.\b/gi,
+		value: 'javascript.'
```

Comment:
What happens if someone writes `The best module for javascript.`? We do want to an error to be reported in this case, though it's better to miss some issues than to have false positives.

Reply:
Yeah, this isn't perfect logic, but it may be the best we can hope for without getting too deep into special casing things.

Reply:
`The best module for javascript.` is not matched by this regex.